tests/functional/aws-node-sdk/test/multipleBackend/get/get.js

```diff
@@ -279,20 +272,6 @@ describe('Multiple backend get object', function testSuite() {
                         done();
                     });
             });
-            it('should return an error on get done to object deleted from AWS',
```

Comment:
shouldn't we still get an error but returned from your head check?

Reply:
I just happened to move this test here: https://github.com/scality/S3/pull/913/files#diff-d1e4ee25cab34d7de04a5e66ff66e53bR346